packages/backend/src/services/OrganizationService/OrganizationService.ts

```diff
@@ -110,7 +110,7 @@ export class OrganizationService {
     }
 
     async updateOrg(
-        { organizationUuid, organizationName, userUuid, ability }: SessionUser,
```

Comment:
unused?

Reply:
Correct, though in retrospect I'll re-introduce this and tag it as unused instead, to keep with my own rule of not modifying implementation too much.